pipeline.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
I may be confusing myself... is this the same file as `./buildkite/pipeline.yml`?